{{cookiecutter.project_slug}}/.github/workflows/memote.yml

```diff
@@ -0,0 +1,30 @@
+name: memote
+
+on:
+  push:
+    branches:
+    - master
```

Comment:
Can we auto-create a `main` branch rather than master?

Reply:
I'm in favor, but it requires coordination with the memote repository, since "master" is hardcoded in the tests explicitly ([here](https://github.com/opencobra/memote/pull/729/files#diff-7f1c01284e89542aea04198b5617298fea93ab5d30222818e6655b5ea2160d48L220) and [here](https://github.com/opencobra/memote/blob/develop/tests/test_for_suite/test_for_reporting/test_for_history.py#L73)) changing also the data for the [mock repo](https://github.com/opencobra/memote/blob/develop/tests/data/memote-mock-repo.tar.gz). Do you prefer it now and in https://github.com/opencobra/memote/pull/729 or in a different pair of PRs?

Reply:
Good question. I guess, it would be better to not completely hardcode it so we have a quick way to change it. Let's tackle it afterwards.